code/modules/mob/living/silicon/robot/robot_model.dm

```diff
@@ -808,27 +808,23 @@
 		/obj/item/borg/apparatus/beaker/service,
 		/obj/item/reagent_containers/cup/beaker/large, //I know a shaker is more appropiate but this is for ease of identification
 		//Skyrat Edit Start: Borg Buff
-		//obj/item/reagent_containers/condiment/enzyme, //edit
-		/obj/item/reagent_containers/condiment/enzyme,
+		//obj/item/reagent_containers/condiment/enzyme, //edit - Borg shaker has it
+		/obj/item/borg/apparatus/beaker, // SR Edit: allows the pickup of different beakers for easier drink mixing
 		/obj/item/reagent_containers/dropper,
 		/obj/item/rsf,
 		/obj/item/storage/bag/tray,
+		/obj/item/storage/bag/tray,
```

Comment:
Do you mean to have this twice?

Reply:
It was requested by about four different people to leave the second tray, currently it was just for some reason in a different space
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/22140677/ebd22d79-93c3-4338-be66-db06e715dcd2)